service/src/integrationTest/kotlin/fi/espoo/evaka/invoicing/service/VoucherValueDecisionGeneratorIntegrationTest.kt

```diff
@@ -1421,6 +1424,8 @@ class VoucherValueDecisionGeneratorIntegrationTest : FullApplicationTest(resetDb
     ) {
         db.transaction { tx ->
             tx.insertAssistanceNeedVoucherCoefficient(
+                AuthenticatedUser.SystemInternalUser,
+                HelsinkiDateTime.now(),
```

Comment:
using (mock)clock.now() is generally better since the test is deterministic and cannot break because of execution time